sigmastate/src/main/scala/sigmastate/interpreter/Interpreter.scala

```diff
@@ -112,6 +146,18 @@ trait Interpreter extends ScorexLogging {
     (res, currContext.value)
   }
 
+  // TODO after HF: merge with old version (`applyDeserializeContext`)
+  /** Same as applyDeserializeContext, but returns SigmaPropValue instead of BoolValue. */
+  def applyDeserializeContextJITC(context: CTX, exp: Value[SType]): (SigmaPropValue, CTX) = {
```

Comment:
I think it is worth to do a test for the method

Reply:
what kind of tests? It is already executed in all existing tests as part of fullReduction, so basically everywhere. And the result is compared against the old AOT version of interpreter.